client/webserver/site/src/css/main_dark.scss

```diff
@@ -132,4 +132,18 @@ body.dark {
   #loader {
     background-color: #13202b77;
   }
+
+  .form-check-input[type=checkbox] {
+    background-color: black;
+
+    &:checked {
+      background-color: #27278d;
+    }
+  }
+
+  .form-check-label {
+    &:hover {
+      font-weight: bold;
+    }
```

Comment:
Styling from **main.scss** still applies. This isn't needed.

Reply:
Oh yeah, right, removed.